lib/msal-react/docs/hooks.md

```diff
@@ -121,7 +119,21 @@ Docs for the APIs `PublicClientApplication` exposes can be found in the `@azure/
 
 ## `useMsalAuthentication` hook
 
-The `useMsalAuthentication` hook will initiate a login if a user is not already signed in. It accepts an `interactionType` ("Popup", "Redirect", or "Silent") and optionally accepts a [request object](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/request-response-object.md#request-and-response-objects) and an `accountIdentifiers` object if you would like to ensure a specific user is signed in. The hook will return the `response` or `error` from the login call and the `login` callback which can be used to retry a failed login.
+The `useMsalAuthentication` hook will initiate a login if a user is not already signed in, otherwise it will attempt to acquire a token.
+
+### Input Parameters
+
+- `interactionType` ("Popup", "Redirect", or "Silent") specifies how you would like to acquire tokens or login when interaction is required (note the "Silent" option has some extra considerations explained below)
```

Comment:
Should we link to the typedocs here?

Reply:
And show using the enum

Reply:
Updated with links to the typedocs. The examples below show using the enum